workflow/rocoto/gfs_tasks.py

```diff
@@ -563,21 +563,21 @@ def aeroanlfinal(self):
 
         return task
 
-    def prepsnowobs(self):
+    def prepsnowcover(self):
 
         deps = []
         dep_dict = {'type': 'task', 'name': f'{self.run}_prep'}
         deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep=deps)
 
-        resources = self.get_resource('prepsnowobs')
-        task_name = f'{self.run}_prepsnowobs'
+        resources = self.get_resource('prepsnowcover')
+        task_name = f'{self.run}_prepsnowcover'
         task_dict = {'task_name': task_name,
                      'resources': resources,
                      'dependency': dependencies,
                      'envars': self.envars,
-                     'cycledef': self.run.replace('enkf', ''),
-                     'command': f'{self.HOMEgfs}/jobs/rocoto/prepsnowobs.sh',
+                     'cycledef': 'gdas_prep_snocvr',
```

Comment:
Should this return to the following?

```suggestion
                     'cycledef': self.run.replace('enkf', ''),
```

Reply:
I don't think so, because the `gdas_prepsnowcover` job is just working in the `00z` cycle.

Reply:
I compared the `gdas_prepsnowcover` task between your XML and my XML, and found they were same. I will set up the experiments again, and rerun my tests.